src/analysis/special_functions/trigonometric.lean

```diff
@@ -2377,6 +2378,18 @@ by rw [log, exp_add_mul_I, ← of_real_sin, sin_arg, ← of_real_cos, cos_arg hx
   mul_div_cancel' _ (of_real_ne_zero.2 (mt abs_eq_zero.1 hx)), ← mul_assoc,
   mul_div_cancel' _ (of_real_ne_zero.2 (mt abs_eq_zero.1 hx)), re_add_im]
 
+lemma exp_map_circle_surjective : function.surjective exp_map_circle :=
```

Comment:
Wouldn't it be easier and more useful to show by first showing that `complex.arg ∘ coe` is a right-inverse, then using `function.right_inverse.surjective`?